src/scss/alert/_inner.scss

```diff
@@ -9,38 +9,24 @@
 @mixin oMessageAlertInner($class) {
 	.#{$class}--alert-inner {
 		.#{$class}__container {
-			@include oMessageAlertInnerContainer();
+			@extend %o-message-inner-alert-notice-container;
+		}
 
-			.#{$class}__content {
-				@include oMessageAlertInnerContent();
-			}
+		.#{$class}__content {
+			@extend %o-message-inner-alert-notice-content;
+			padding-left: $_o-message-alert-icon-size * 1px;
 
-			.#{$class}__actions {
-				@include oTypographyPadding($bottom: 3);
+			&-main {
+				@extend %o-message-inner-alert-notice-content-main ;
 			}
 
-			.#{$class}__highlight,
-			.#{$class}__additional-info {
-				@include oTypographyMargin($top: 0);
+			&-highlight {
+				@include oMessageContentHighlight;
 			}
 		}
-	}
-}
-
-// Alert Inner message container styles
-// Outputs container layout
 
-@mixin oMessageAlertInnerContainer () {
-	@include oMessageContainer();
-	padding: oTypographySpacingSize(2) oTypographySpacingSize(5) oTypographySpacingSize(2) oTypographySpacingSize(4);
-};
-
-
-// Alert Inner message content styles
-// Outputs content layout
-
-@mixin oMessageAlertInnerContent () {
```

Comment:
Is this removed? Should go in the migration guide if so.

Reply:
Same with `oMessageAlertInnerContainer` etc.

Reply:
:+1: